crates/forge/tests/it/zk/cheats.rs

```diff
@@ -210,6 +214,184 @@ forgetest_async!(test_zk_use_factory_dep, |prj, cmd| {
     ).await;
 });
 
+forgetest_async!(test_zk_broadcast_raw_create2_deployer, |prj, cmd| {
```

Comment:
rename the tests to be more descriptive. also if this is a foundry script it should start with `script_zk` instead of `test_zk` (or at least that's what we have been doing so far)

Reply:
I also see that below you are checking the broadcast output, as part of a script, but here you aren't really checking anything other than "it runs".
What I meant was a forge test `contract ZkBroadcastRaw is Test` and in that test function you'd check the observable of what was executed.

For example:
```
Counter counter = Counter(0x...);
uint256 initial = counter.count();
vm.broadcastRawTransaction(...);
assertEq(initial + 1, counter.count());
```

Reply:
This is a `script`, so we don't have `assertEq`, unfortunately. 
I checked the values calling the function on a different command, so I can eventually assert at the end

Reply:
Hence `contract ZkBroadcastRaw is Test`, but yeah we discussed offline, I think it's a bit clearer now what we would like for the tests to look like?

Reply:
Yup, I think require is the way to go 💪 
Please take a look and let me know